onnxruntime/core/providers/nnapi/nnapi_provider_factory.cc

```diff
@@ -7,24 +7,27 @@
 using namespace onnxruntime;
 
 namespace onnxruntime {
+static_assert(NNAPI_FLAG_MAX < 32, "We can have at most 32 (0-31) nnapi flags");
 
```

Comment:
Can this be where the flags are defined or does it have to be in a .cc?

Reply:
This seems not really necessary after using bit flags, removed